docs/platform/howto/byoc/assign-project-custom-cloud.md

```diff
@@ -1,48 +1,42 @@
-
-:::important
-Custom cloud configuration in Aiven is an
-[early availability feature](/docs/platform/concepts/beta_services). You cover the costs associated with building and
-maintaining your custom cloud payments for your integrated AWS
-infrastructure and Aiven services within the custom cloud.
-:::
+Configure the availability of your [custom cloud](/docs/platform/concepts/byoc) to access
+it in all the projects, selected organizational units, or specific projects only.
 
 ## About making custom clouds available from your projects
 
 With the BYOC feature enabled, you can
-[create custom clouds](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven organization. As a part of the
-[initial custom cloud's setup in Aiven Console](/docs/platform/howto/byoc/create-custom-cloud), you select in what projects you'll be able to use your new
-custom cloud to create services and make your
-cloud available for all the projects in your organization, selected
-organizational units, or specific projects only.
-
-Later, you can come back to the **Available projects** tab in your
+[create custom clouds](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven
+organization. As a part of the
+[initial custom cloud's setup in the Aiven Console](/docs/platform/howto/byoc/create-custom-cloud),
+you select in what projects you'll be able to use your new custom cloud to host Aiven
+services. Later, you can come back to the **Available projects** tab in your
 cloud's page in [Aiven Console](https://console.aiven.io/) and update
 the settings you configured during the
 [initial custom cloud's setup](/docs/platform/howto/byoc/create-custom-cloud).
 
 1.  In the **Custom cloud's availability in your organization**
     section, select either:
-    - **By default for all projects** to make your custom cloud
-      available in all existing and future projects in the
-      organization, or;
-    - **By selection** to pick specific projects or organizational
-      units where you want your custom cloud to be available.
+    -   **By default for all projects** to make your custom cloud
+        available in all existing and future projects in the
+        organization, or;
+    -   **By selection** to pick specific projects or organizational
+        units where you want your custom cloud to be available.
 1.  If you go for the **By selection** option, the **Assign
     organizational units** field and the **Assign projects** field show
     up. Enter the names of organizational units and/ or projects in
     which you want to be able to use your custom cloud.
 
 ## Prerequisites
 
--  Administrator's role for your Aiven organization
--  At least one
-   [custom cloud created](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven organization
--  Access to [Aiven Console](https://console.aiven.io/)
+-   Administrator's role for your Aiven organization
```

Comment:
Sales just informed me that the Priority Support Tier is also a prerequisite for BYOC